controllers/module_reconciler.go

```diff
@@ -130,26 +131,26 @@ func (r *ModuleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return res, fmt.Errorf("could get DaemonSets for module %s: %v", mod.Name, err)
 	}
 
-	for kernelVersion, m := range mappings {
-		completedSuccessfully, err := r.handleBuild(ctx, mod, m, kernelVersion)
+	for kernelVersion, mld := range mldMappings {
+		completedSuccessfully, err := r.handleBuild(ctx, mld, kernelVersion)
 		if err != nil {
 			return res, fmt.Errorf("failed to handle build for kernel version %s: %v", kernelVersion, err)
 		}
 		if !completedSuccessfully {
-			logger.Info("Build has not finished successfully yet:skipping handling signing and driver container for now", "kernelVersion", kernelVersion, "image", m)
+			logger.Info("Build has not finished successfully yet:skipping handling signing and driver container for now", "kernelVersion", kernelVersion, "mld", mld)
```

Comment:
How about adding `mld` to the logger instead of manually adding it everywhere?

Reply:
fixed